tests/System.Data.SQLite.Tests/System.Data.SQLite.Tests.csproj

```diff
@@ -5,11 +5,11 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(Platform)' != 'win-x64' and '$(Platform)' != 'win-x86' and '$(Platform)' != 'osx-x64' ">
-    <TargetFrameworks>netcoreapp3.1;net472;xamarin.ios10;monoandroid81</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net472;net5.0;xamarin.ios10;monoandroid81</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(Platform)' == 'win-x64' ">
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net472;net5.0</TargetFrameworks>
```

Comment:
I've been targeting `net6.0` in test projects since it can be run on macOS arm machines, though I'm not sure if appveyor support net6 or arm on macOS.

Reply:
I started with that, but got build errors because the toolchain is currently set up for VS2019, which doesn't support it. Decided to just go with `net5.0` for now.